plugins/com.google.cloud.tools.eclipse.appengine.newproject/plugin.properties

```diff
@@ -7,5 +7,10 @@ gcp.category = Google Cloud Platform
 standard.wizard.name = Google App Engine Standard Java Project
 standard.wizard.description = Creates an App Engine Standard Java project.
 
-flex.wizard.name=Google App Engine Flexible Java Project
-flex.wizard.description=Creates an App Engine Flexible Java project.
+flex.war.wizard.name=Google App Engine Flexible Java WAR Project
+flex.war.wizard.description=Creates an App Engine Flexible Java project whose deploy artifact is a \
+ web application archive (WAR).
```

Comment:
WAR file; don't spell out web application archive; and you know;  let's focus on the runtime here; not the deploy artifact. I.e. something like

"Creates an App Engine Flexible Java project for the Jetty runtime"

Reply:
But we have a Tomcat runtime too. Deployable with "gcloud beta": https://github.com/GoogleCloudPlatform/tomcat-runtime#cloud-sdk.

Reply:
Perhaps "the Jetty/Tomcat runtime" will do.